pkg/apis/pipeline/v1beta1/pipeline_types_test.go

```diff
@@ -337,9 +337,16 @@ func TestPipelineTask_ValidateRegularTask_Failure(t *testing.T) {
 		},
 		expectedError: *apis.ErrDisallowedFields("taskref.resolver"),
 	}, {
-		name: "pipeline task - use of resource without the feature flag set",
+		name: "pipeline task - use of resolver params without the feature flag set",
 		task: PipelineTask{
-			TaskRef: &TaskRef{Name: "boo", ResolverRef: ResolverRef{Resource: []ResolverParam{{}}}},
+			TaskRef: &TaskRef{Name: "boo", ResolverRef: ResolverRef{Params: []Param{{
+				Name: "bar",
+				Value: ArrayOrString{
```

Comment:
`ArrayOrString` was renamed to `ParamValue` in https://github.com/tektoncd/pipeline/pull/5304 -- we need to rename it in this PR as well